galleon-pack/galleon-shared/src/main/resources/layers/standalone/microprofile-lra-participant/layer-spec.xml

```diff
@@ -25,6 +25,7 @@
     <dependencies>
         <layer name="cdi"/>
         <layer name="jaxrs"/>
+        <layer name="microprofile-config"/>
```

Comment:
@xstefank Can this layer function properly if the microprofile-config-smallrye subsystem is not part of the standalone.xml configuration and the user doesn't try to configure mp.lra.propagation.active?

If yes adding 'optional="true"' to this is a possibility as it lets the user exclude that layer and thus not have the subsystem in their config. If it won't work properly this definitely needs to be there.

Even if it won't work properly it's valid for you to decide that you always want it added. (You could always choose to make it optional later; otoh making it required later can be a compatibility problem.) I just want to make sure this PR is doing what you intend.